.circleci/config.yml

```diff
@@ -4131,7 +4131,7 @@ jobs:
             git clean -fd
             # Loop to prevent failure due to maven-ant-tasks not downloading a jar..
             for x in $(seq 1 3); do
-                ${ANT_HOME}/bin/ant realclean; ${ANT_HOME}/bin/ant jar dtest-jar
+                ${ANT_HOME}/bin/ant realclean; ${ANT_HOME}/bin/ant jar dtest-jar -Dno-checkstyle=true -Dno-check=true
```

Comment:
it shouldn't be necessary to supply both `-Dno-checkstyle=true -Dno-check=true`

i.e. `-Dno-check=true` should imply `-Dno-checkstyle=true -Drat.skip=true -Declipse-warnings.skip=true -Ddependency-check.skip=true`

here's a patch i was playing with before… 
https://github.com/apache/cassandra/compare/trunk...thelastpickle:cassandra:mck/buildfiles-linter-docs/4.1

( that patch also allows you to specify there shorter form of `-Dno-linter=` )

Reply:
This is for older branches - are we going to apply those changes to those older branches as well?

Reply:
erh, no. go nuts then.